### PR TITLE
Update ISSUE_TEMPLATE.md

### LINKED ISSUE #10907: Update ISSUE_TEMPLATE.md
**Describe the issue you encountered:** Update ISSUE_TEMPLATE.md.

We could use https://github.com/mochajs/mocha/blame/master/.github/ISSUE_TEMPLATE.md as a reference.

https://github.com/mochajs/mocha/pull/2869

## td;dr

````
<!--
Have you searched for similar issues before submitting this one? There is a lot of feedbacks and bug reports we have received and closed as a duplicate. By using search engines along with GitHub search function you would be able to find duplicates more effectively. Also please visit our community site for common issues: https://community.brave.com/
-->

### Brave Version
<!--
Please open about:brave, copy the version information, and paste it.
-->

### Description
<!--
[Description of the issue]
-->

### Steps to Reproduce
<!--
Please add a series of steps to reproduce the problem. See https://stackoverflow.com/help/mcve for in depth information on how to create a mimimal, complete, and verifiable example.
-->

  1.
  2.
  3.
    
**Actual result:**
<!--
Add screenshots if needed
-->

**Expected result:**

**Reproduces how often:** [What percentage of the time does it reproduce?]

### Additional Information
<!--
Place an `x` between the square brackets on the lines below if applied
-->
- [ ] Will the steps above reproduce in a fresh profile?
- [ ] Is this an issue in the currently released version?
- [ ] Can this issue be consistently reproduced?

<!--
Any additional information, related issues, configuration or data that might be necessary to reproduce the issue.
-->
````

----

I. Replace `Did you search for similar issues before submitting this one?` with a comment block eg.

````
<!--
Have you searched for similar issues before submitting this one? There is a lot of feedbacks and bug reports we have received and closed as a duplicate. By using search engines along with GitHub search function you would be able to find duplicates more effectively. Also please visit our community site for common issues: https://community.brave.com/
-->
````

II. Replace listing with `h3` header

III. Ask submiters to paste info available on about:brave 

````
### Brave Version
<!--
Please open about:brave, copy the version information, and paste it.
-->
````

IV. Replace `Describe the issue you encountered:` with

````
### Description
<!--
[Description of the issue]
-->
````

V. Replace STR into this block, following the mocha's template:

````
### Steps to Reproduce
<!--
Please add a series of steps to reproduce the problem. See https://stackoverflow.com/help/mcve for in depth information on how to create a mimimal, complete, and verifiable example.
-->

  1.
  2.
  3.
    
**Actual result:**
<!--
Add screenshots if needed
-->

**Expected result:**

````

### PR DESCRIPTION
**Pull request by luixxiul**
- Add comment blocks
- Replace list items with headers

Closes #10907
Based on https://github.com/mochajs/mocha/pull/2869

Auditors:

Test Plan:

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Test Plan:

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,31 +1,41 @@
-- Did you search for similar issues before submitting this one?
+<!--
+Before submitting this issue, please visit our wiki for common ones: https://github.com/brave/browser-laptop/wiki
 
-- Describe the issue you encountered:
+Have you searched for similar issues? There is a lot of feedbacks and bug reports we have received and closed as duplicate.
+By using search engines along with GitHub search function you would be able to find duplicates more efficiently.
 
-- Platform (Win7, 8, 10? macOS? Linux distro?):
+For more, check out our community site: https://community.brave.com/
```
Comment (luixxiul) [inline .github/ISSUE_TEMPLATE.md]: I was going to add a link to our discord channel, which on the second thought seemed too much. If it isn't, I'll add that.

Comment (srirambv) [reply .github/ISSUE_TEMPLATE.md]: Not sure if the first line about visiting wiki makes sense at top.  I would have it like this 

```
Have you searched for similar issues? There is a lot of feedback and bug reports that we have received and closed as duplicate.  
Please search for existing issue via GitHub search or using a search engines. 

Please check our community site: https://communit.brave.com and also have a look at our Wiki: https://github.com/brave/browser-laptop/wiki/
```

Comment (luixxiul) [reply .github/ISSUE_TEMPLATE.md]: > Not sure if the first line about visiting wiki makes sense at top.

I'd agree. I'll edit this comment block.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,31 +1,40 @@
...
-
-- Screenshot if needed:
-
-- Any related issues:
+<!--
+Have you searched for similar issues? There is a lot of feedbacks and bug reports we have received and closed as duplicate.
+Before submitting this issue, please visit our wiki for common ones: https://github.com/brave/browser-laptop/wiki
+By using search engines along with GitHub search function, you would be able to find duplicates more efficiently.
+
+For more, check out our community site: https://community.brave.com/
```
Comment (luixxiul) [inline .github/ISSUE_TEMPLATE.md]: `Our Discord community chat is available too: https://discordapp.com/invite/k57tYrS`

**Comment by bbondy** [comment]
I think we'll want to tweak it, but I'm ok with this as the new starting point so merging.

**Pull request merged**